package.json

```diff
@@ -70,7 +70,7 @@
     "@commitlint/config-conventional": "^11.0.0",
     "@cypress/react": "^5.12.0",
     "@cypress/webpack-dev-server": "^1.8.4",
-    "@sage/design-tokens": "^2.3.0",
+    "@sage/design-tokens": "^2.23.0",
```

Comment:
We'll need to update the peerDependencies above with the same thing

Reply:
I'm just curious, why do we have these in peer-dependencies as well?

Reply:
This package provides the design tokens, so if users don't have it installed then our components will look for the Design Token CSS variables and they won't be there. So users need to have this installed as well 👍🏻

Reply:
We're compatible with any version `>2.3.0, <3.0.0` so we don't need to update the peer dependency.